pkg/deploy/controller/deploymentconfig/factory.go

```diff
@@ -40,7 +40,7 @@ func (factory *DeploymentConfigControllerFactory) Create() controller.RunnableCo
 		},
 	}
 	queue := cache.NewFIFO(cache.MetaNamespaceKeyFunc)
-	cache.NewReflector(deploymentConfigLW, &deployapi.DeploymentConfig{}, queue, 2*time.Minute).Run()
+	cache.NewReflector(deploymentConfigLW, &deployapi.DeploymentConfig{}, queue, 30*time.Second).Run()
```

Comment:
That really depends on what is needed.:) The reflector itself will not exit on a resync notification any more.  It will break the watch, call Resync on the store object the reflector was configured to use, then initiate the watch.

However, the DeltaFIFO was the only store implementation that had access to the local cache and a notification means to allow the resync to cause the local cache to be inspected.  Other storage types will do nothing on a resync or just ensure that all items in the fifo are in the processing queue for the workers to process.

So, while the reflector will not exit on a resync call any more, I would say only the informer (or reflectors configured similarly to the informer) will actually cause an evaluation of the local cache.  Whether or not that is a big deal depends on what the use case is.